File: pyg_interface/src/main/java/cn/itcast/core/service/SeckillOrderService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.entity.PageBean;
import cn.itcast.core.pojo.entity.PageResult;
import cn.itcast.core.pojo.seckill.SeckillOrder;

public interface SeckillOrderService {
    PageResult search(SeckillOrder seckillOrder, Integer page, Integer rows);

    //手动分页查询
    PageBean<SeckillOrder> findPage(String userId, Integer pageNum, Integer pageSize);

    //取消订单
    void cancel(Long id);
}

File: web_user/src/main/webapp/js/service/seckillOrderService.js
//服务层
app.service('seckillOrderService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../seckillOrder/findAll.do');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../seckillOrder/findPage.do?page='+page+'&rows='+rows);
	}

	//取消订单
    this.cancel=function(id){
        return $http.get('../seckillOrder/cancel.do?id='+id);
    }

});
